TCAT/Controllers/FavoritesViewController.swift

```diff
@@ -127,11 +197,22 @@ extension FavoritesViewController: UICollectionViewDelegateFlowLayout {
 extension FavoritesViewController: PulleyDrawerViewControllerDelegate {
 
     func partialRevealDrawerHeight(bottomSafeArea: CGFloat) -> CGFloat {
-        return 200
+        return 192 + bottomSafeArea
+    }
+
+    func collapsedDrawerHeight(bottomSafeArea: CGFloat) -> CGFloat {
+        return 54 + bottomSafeArea
     }
 
     func supportedDrawerPositions() -> [PulleyPosition] {
         return [.collapsed, .partiallyRevealed]
     }
 
+    func drawerChangedDistanceFromBottom(drawer: PulleyViewController, distance: CGFloat, bottomSafeArea: CGFloat) {
+        let totalHeight : CGFloat = 192 - 54
```

Comment:
Can we get these constants from a common source in case we decide to change them in one place but forget to change them here?

Reply:
+1